ckcp/openshift_dev_setup.sh

```diff
@@ -76,7 +77,7 @@ parse_args() {
 }
 
 init() {
-  APP_LIST="openshift-gitops ckcp compute"
+  APP_LIST="openshift-gitops compute ckcp register-compute"
```

Comment:
Don't see any function called 'register-compute'.
Also, I see two other functions install_compute and install_register_compute.
- Please rename install_register_compute to register_compute.
- Need to include install_compute in the APP_LIST too.

Reply:
@bnallapeta `main()` is processing APP_LIST and generating func names that it's then executing. So APP_LIST looks good.

Reply:
Resolving this as I believe the APP_LIST is ok. Please let me know if it's not.